learningresources/tests/test_api.py

```diff
@@ -117,6 +129,11 @@ def test_get_repo(self):
             self.user_norepo.id
         )
 
+    def test_invalid_create_repo(self):
+        """Create a repository with empty name and description"""
+        self.assertRaises(IntegrityError,
+                          lambda: create_repo(None, None, self.user.id))
+
```

Comment:
Same comments on checking the value of the exception, and not having the API raise integrity errors to the caller of the API. I won't repeat this comment again for other occurrences, so I'm not too annoying and also since a decision needs to be made about adding a `ValidationError`.